### DISCUSSION TITLE
unlike/remove likes after download

### DISCUSSION THREAD

**Discussion by ZeroThreeQuarter** [TumblThreeApp/TumblThree#353] [Ideas]
Is there an easy way to remove all the "likes" once they've been downloaded for archiving? 
Or does anyone have a suggestion on how to remove a bunch of them at a time?

If not, then is there a way to request this feature? hehe

Big thanks

**Comment by thomas694** [comment]
You can't do it with TumblThree and I don't know any tool or browser extension for that.
But maybe someone else knows something like that.

I moved it to "Ideas".
If someone else has a similar use case or likes that idea, comment or give it an arrow/thumb up.

**Reply by hrxn** [reply]
Well, it could be possible, at least there are some API endpoints that should do it.
https://www.tumblr.com/docs/en/api/v2#userlike--like-a-post
https://www.tumblr.com/docs/en/api/v2#userunlike--unlike-a-post

I've done a quick and dirty test with https://github.com/nostalgebraist/pytumblr2, and while this is a bit flaky (because this probably has not been updated to handle the latest changes in the API response format, I think?), but error messages notwithstanding, I still could like and unlike some random posts.

**Reply by thomas694** [reply]
Thanks for your help and researching it. As we don't use API v2 directly, we can't use these endpoints itself.
But maybe we can find another way, if many users are interested in this idea.

**Comment by ZeroThreeQuarter** [comment]
If there was an option to "unlike when downloaded" that'd be baller! :D
